frontend/components/LanguageSelector.vue

```diff
@@ -12,15 +12,19 @@ const bmmLanguages = ref<string[]>([]);
 
 const api = useAPI();
 
-watch([() => props.env, () => props.languages], async([newEnv, newLanguages])=> {
-  if (newLanguages.length > 0){
-    bmmLanguages.value = newLanguages;
-  } else {
-    bmmLanguages.value = (await api.getLanguages({environment: newEnv})).Languages;
-  }
-  console.log("selected language should be: ", bmmLanguages.value[0]);
-}, {immediate: true});
-
+watch(
+    [() => props.env, () => props.languages],
+    async ([newEnv, newLanguages]) => {
+        if (newLanguages.length > 0) {
+            bmmLanguages.value = newLanguages;
+        } else {
+            bmmLanguages.value = (
+                await api.getLanguages({ environment: newEnv })
```

Comment:
really dont like the formatting. i tried to tweak prettier but couldnt figure it out.
not important for the pr